Website/src/rendering/src/services/AuthenticationService.ts

```diff
@@ -15,7 +15,12 @@ export const isAuthenticationEnabled =
 
 // build up url for openid connect so user can log into ordercloud via auth0
 const roles = orderCloudScope.join(' ');
-export const loginUrl = `${ORDERCLOUD_BASE_API_URL}/ocrplogin?id=${ORDERCLOUD_OPENID_CONNECT_ID}&cid=${ORDERCLOUD_BUYER_CLIENT_ID}&roles=${roles}`;
+export const loginUrl = (appstartpath = ''): string => {
```

Comment:
Can the name please be changed to `getLoginUrl` as it is a function now. Thanks.